compiler-rt/test/nsan/stable_sort.cpp

```diff
@@ -0,0 +1,55 @@
+// RUN: %clangxx_nsan -O2 -g %s -o %t
+// RUN: %run %t 2>&1 | FileCheck %s
+
+// RUN: %clangxx_nsan  -fno-builtin -O2 -g %s -o %t
```

Comment:
excess space before -fno-builtin

a comment would be useful why -fno-builtin is added

Reply:
done